lib/workers/repository/process/common.ts

```diff
@@ -0,0 +1,8 @@
+import type { RenovateConfig } from '../../../config/types';
+
+export function baseBranchesEqualsDefault({
+  baseBranches,
+  defaultBranch,
+}: RenovateConfig): boolean {
+  return baseBranches?.toString() === [defaultBranch].toString();
```

Comment:
no, log when `baseBranch !== defaultBranch`

Reply:
just to confirm, if `baseBranches = ['main','dev']` and `defaultBranch='main'` we only log when processing `dev`? wouldn't it be better to explicitly log `main` in this case?

one thing to note is that `baseBranch` is added to the meta only in the extraction and writing stages.
that is only if `baseBranches!=[defaultBranch]`

Reply:
I think if there's more than one base branch then we should log the baseBranch, even when it's the default such as `main`. If there's only one base branch and it's not default, maybe log too, or no need?

Reply:
☝️

Reply:
> I think if there's more than one base branch then we should log the baseBranch, even when it's the default such as main. If there's only one base branch and it's not default, maybe log too

^^ thats exactly what is done right now,

```ts
baseBranches?.toString() === [defaultBranch].toString()
```
this will be evaluated as truth if and only if `baseBranches[0] === defaultBranch && baseBranches.length === 1`

`Array.prototype.toString` uses `Array.prototype.join(',')` behind the scenes -
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/toString#description

so i was basically waiting for a decision to be made regarding -
> maybe log too, or no need?